WcaOnRails/config/locales/de.yml

```diff
@@ -2365,17 +2371,17 @@ de:
           Organisatoren. Du kannst den Status deiner Registrierung auch %{here}
           einsehen.
       accepted:
-        #original_hash: 34515b5
-        mail_subject: 'Deine Registrierung für %{comp_name} wurde bestätigt.'
+        #original_hash: 6f1fa1f
+        mail_subject: 'Anmeldung Akzeptiert: %{comp_name}'
         #original_hash: 24d2883
         message_html: >-
           Deine Registrierung für %{comp_name} wurde bestätigt. Wie immer kannst
           du den Status deiner Registrierung %{here} ansehen.
         #original_hash: 1852cbf
         see_you_soon: 'Wir freuen uns darauf dich bei %{comp_name} zu sehen!'
       pending:
-        #original_hash: e48d8bd
-        mail_subject: 'Du wurdest auf die Warteliste für die %{comp_name} gesetzt.'
+        #original_hash: 8d26297
+        mail_subject: 'Anmeldung auf die Warteliste gesetzt: %{comp_name}'
```

Comment:
In den anderen Fällen oben machst du Title Case, d.h. du schreibst Worte groß die eigentlich nicht groß geschrieben werden. Dann solltest du das hier denke ich auch tun, zumindest für alles außer Konjunktionen und Artikel.